### DISCUSSION TITLE
Nested create error

### DISCUSSION THREAD

**Discussion by srisub1** [prisma/prisma#2079] [Q&A] [answered]
I upgraded to 2.0.0.beta.1, and an existing mutation with nested create just stopped working. I get this very verbose but incomprehensible error:
Response: {"data":null,"errors":[{"message":"\nInvalid `prisma.user.create()` invocation in\n/Users/.../User/src/types/User.ts:143:66\n\nError occurred during query execution:\nInterpretationError(\"Error for binding \\'1\\': DomainError(ConversionFailure(\\\"record identifier\\\", \\\"assimilated record identifier\\\"))\")","locations":[{"line":66,"column":3}],"path":["login"]}]}

----
My resolver has code that looks like this:
```
 ctx.prisma.user.create({ data: {
		email: inputData.email,
		firstName: inputData.firstName,
		lastName: inputData.lastName,
		info: {
			create: {
			}
		},
		contact: {
			create: {
			}
		}
})
```
The main model "User" has two non-optional model fields, that i am trying to create as nested creates.

Please help! Do i need to go back to preview022, which used to work for this,

**Comment by janpio** [comment]
The release notes of beta 1 explain what changed in the relation syntax: https://github.com/prisma/prisma/releases/tag/2.0.0-beta.1 After you fix that, it should work again.

**Comment by srisub1** [comment, marked as answer]
Thanks for that tip. I cleaned up node_modules, and reinstalled beta.1, but now i get this mysterious error during nexus-prisma makeSchema.

Failed to import prisma client package at @prisma/client. The following error occured while trying:
        SyntaxError: Unexpected token 
 in JSON at position 502
    at JSON.parse (<anonymous>)
    at Object.<anonymous> (/../Default/node_modules/@prisma/client/index.js:703:19)
    at Module._compile (internal/modules/cjs/loader.js:1147:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)
    at Module.load (internal/modules/cjs/loader.js:996:32)
    at Function.Module._load (internal/modules/cjs/loader.js:896:14)
    at Module.require (internal/modules/cjs/loader.js:1036:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.exports.getPhotonDmmf (/../Default/node_modules/nexus-prisma/src/dmmf/utils.ts:8:12)
    at Object.exports.getTransformedDmmf (/../Default/node_modules/nexus-prisma/src/dmmf/transformer.ts:19:30)

**Reply by janpio** [reply]
Can you open an issue please? We unfortunately convert this discussion into a issue and it would be much easier to discuss this and bring to a conclusion there. Thanks!

**Reply by srisub1** [reply]
Create bug: https://github.com/prisma/prisma/issues/2084. Empty comments in schema file seems to have caused the problem.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
